src/services/AppStateManager.tsx

```diff
@@ -20,8 +20,7 @@ export class AppStateManager extends PureComponent<Props, State> {
 
   handleAppStateChange = (nextAppState: string) => {
     const { appState } = this.state;
-    console.log({ appState, nextAppState });
-    if (appState !== 'active' && nextAppState === 'active') {
+    if (appState === 'background' && nextAppState === 'active') {
```

Comment:
Is it working correctly on iOS, as there's third state?

> [iOS] inactive - This is a state that occurs when transitioning between foreground & background, and during periods of inactivity such as entering the Multitasking view or in the event of an incoming call

Reply:
as a workaround - yes. If the inactive state is necessary I will need to find a proper fix